src/data/bucket/line_bucket.js

```diff
@@ -537,6 +541,10 @@ class LineBucket implements Bucket {
             (this.clipStart + (this.clipEnd - this.clipStart) * this.distance / this.totalDistance)  * (MAX_LINE_DISTANCE - 1) :
             this.distance;
     }
+
+    updateDistance(prev: Point, next: Point) {
+        this.distance += prev.dist(next);
+    }
```

Comment:
Nit: since we want to call `updateScaledDistance` after every `updateDistance`, we might as well call it inside `updateDistance` to DRY up the code a little.